90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -977,10 +977,19 @@ populate_be_list() {
       # All other datasets are ignored
       continue
     fi
+    if [ -n "${BOOTFS}" ] && [ "${BOOTFS}" = "${fs}" ] ; then
+      # If BOOTFS is defined, we'll manually append it to the array
```

Comment:
Might need to set `have_bootfs=1` here for some local variable `have_bootfs`, see comment on Line 990.